src/context.hh

```diff
@@ -20,6 +20,7 @@ class KeymapManager;
 class AliasRegistry;
 
 enum Direction { Backward = -1, Forward = 1 };
+enum Velocity { FullScreen = 0, HalfScreen = 1, Line = 2 };
```

Comment:
"Velocity" is defined as "the speed of something in a given direction". I think "distance" would be a better name here.

Reply:
Fair enough, I renamed this. Thanks!